.github/workflows/tests.yml

```diff
@@ -250,6 +250,9 @@ jobs:
       - name: Test (Firefox)
         run: wasm-pack test --headless --firefox --features=js,test-in-browser
       - name: Test (Chrome)
+        # We get spurious failures, see:
+        # https://github.com/rust-random/getrandom/issues/400
```

Comment:
Ideally, we would still detect Chrome failures on working OSes (Linux/macOS). We could either:
  - Not test Chrome on  Windows. We are already running Edge there which is just reskinned Chrome.
  - Update the matrix to just mark Windows as flaky in general (we have Edge and Chrome problems on Windows)

```
matrix:
  include:
    - os: ubuntu-22.04
      host: x86_64-unknown-linux-musl
    - os: windows-2022
      host: x86_64-pc-windows-msvc
      continue-on-error: true
    - os: macos-12
      host: x86_64-apple-darwin
```

Reply:
Yeah, I also [observed](https://github.com/rust-random/getrandom/actions/runs/8569924021/job/23487836088) spurious failures with Firefox on Windows as well, though they are less frequent than with Chrome, so temporarily applying `continue-on-error: true` for the whole Web Windows job sounds like a good idea.

Reply:
Sounds good, we can use #400 to track how to actually fix these tests.